core/trino-main/src/test/java/io/trino/block/TestDictionaryBlock.java

```diff
@@ -353,27 +349,15 @@ private static void assertDictionarySizeMethods(Block block)
 
         boolean[] selectedPositions = new boolean[positions];
         selectedPositions[0] = true;
-        assertEquals(
-                DictionaryBlock.create(allIds.length, block, allIds).getPositionsSizeInBytes(selectedPositions, 1),
-                block.getPositionsSizeInBytes(selectedPositions, 1) + Integer.BYTES);
+        assertThat(DictionaryBlock.create(allIds.length, block, allIds).getPositionsSizeInBytes(selectedPositions, 1)).isEqualTo(block.getPositionsSizeInBytes(selectedPositions, 1) + Integer.BYTES);
```

Comment:
Some manual post-processing to adds line wraps would be nice :)

Reply:
Too much to handle I think.